pkg/common/helper_test.go

```diff
@@ -0,0 +1,167 @@
+package common
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+)
+
+type IsFileTestSuite struct {
+	suite.Suite
+	tempDir string
+}
+
+type IsDirTestSuite struct {
+	suite.Suite
+	tempDir string
+}
+
+type FileExistTestSuite struct {
+	suite.Suite
+	tempDir string
+}
+
+type StringSliceToIntSliceTestSuite struct {
+	suite.Suite
+}
+
+type RetryUntilSuccessfulTestSuite struct {
+	suite.Suite
+}
+
+func (suite *IsFileTestSuite) SetupTest()  {
+	var err error
+
+	// create a temp directory
+	suite.tempDir, err = ioutil.TempDir("", "isfile-test")
+	suite.Require().NoError(err)
+}
+
+func (suite *IsFileTestSuite) TestIsFileFuncPositive() {
```

Comment:
If you made each helper into a suite, no need to have the function name the test method names. Rather than `TestIsFileFuncPositive`, you can do `TestPositive` - you're already in the `IsFile` namespace.